kratos/solving_strategies/strategies/residualbased_linear_strategy.h

```diff
@@ -357,125 +382,216 @@ class ResidualBasedLinearStrategy
 
     double Solve() override
     {
-        KRATOS_TRY
```

Comment:
i would put the KRATOS_TRY and KRATOS_CATCH blocks

Reply:
Ping @RiccardoRossi